packages/contracts/contracts/utils/AppStorage.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "./UnstructuredStorage.sol";
+import "../core/IDAO.sol";
+
+contract AppStorage {
+    using UnstructuredStorage for bytes32;
+    
+    /* Hardcoded constants to save gas
+    bytes32 internal constant DAO_POSITION = keccak256("aragonCore.appStorage.dao");
```

Comment:
How about `core.storage.dao`? It's totally cosmetic, I know. But clean code starts here too :)

Reply:
DONE